specification/summary-care-record.yaml

```diff
@@ -38,43 +38,49 @@ paths:
       responses:
         "200":
           description: OK successful
           content:
             application/fhir+json:
               schema:
                 type: object
+              example: "{\n    \"OperationOutcome\": {\n      \"issue\": {\n        \"severity\": {\n            \"value\": \"information\"\n        },\n        \"code\": {\n            \"value\": \"informational\"\n        },\n        \"diagnostics\": {\n            \"value\": \"Resource has been successfully updated.\"\n        }\n      }\n    }\n}"
+        "400":
+          description: ACL consent not set
+          content:
+            application/fhir+json:
+              schema:
+                type: object
+              example: "{\n    \"OperationOutcome\": {\n      \"issue\": {\n        \"severity\": {\n            \"value\": \"error\"\n        },\n        \"code\": {\n            \"value\": \"invalid\"\n        },\n        \"diagnostics\": {\n            \"value\": \"ttps://fhir.nhs.uk/STU3/ValueSet/Spine-ErrorOrWarningCode-1 == static display value\"\n        }\n      }\n    }\n}"
         "412":
           description: ACL consent not set
           content:
             application/fhir+json:
               schema:
-                type: object                
-        "504":
-          description: Gateway Timeout
+                type: object
+              example: "{\n    \"OperationOutcome\": {\n      \"issue\": {\n        \"severity\": {\n            \"value\": \"error\"\n        },\n        \"code\": {\n            \"value\": \"invalid\"\n        },\n        \"diagnostics\": {\n            \"value\": \"ttps://fhir.nhs.uk/STU3/ValueSet/Spine-ErrorOrWarningCode-1 == static display value\"\n        }\n      }\n    }\n}"
```

Comment:
Use diagnostics value to provide error descriptions of what will be returned

Reply:
it is used, can have a quick call to discuss?